tests/README.md

```diff
@@ -97,26 +97,26 @@ Examining CI run
 ----------------
 * Set BUILD_ID is travis ci's build id. E.g. build id is 169519874 for the following build
 ```bash
-https://travis-ci.org/apache/kafka/builds/169519874
```

Comment:
The travis CI was already removed from kafka. maybe we can remove those docs. @ijuma WDYT?

Reply:
Sounds good.

Reply:
Removed the doc.

Reply:
Agree to remove the unused travis CI doc.